app/Main.hs

```diff
@@ -252,19 +281,33 @@ main = do
   localPath <- maybe (getEnv "HOME") return oLocalPath
   eS3Bucket <- lookupEnv "S3_BUCKET"
   let gitAccount = "lehins" -- "commercialhaskell"
-      delay = fromMaybe 60 oDelay * 100000
+      delay = fromMaybe 60 oDelay * 1000000
       ms3Bucket = msum [oS3Bucket, eS3Bucket]
       gitUser =
         GitUser
         { userName = oUsername
         , userEmail = oEmail
         , userGPG = oGPGSign
         }
+  case (ms3Bucket, oAwsAccessKey, oAwsSecretKey) of
+    (Just _, Just accessKey, Nothing) -> setEnv "AWS_ACCESS_KEY_ID" accessKey
```

Comment:
Yes. Or: don't even bother with command line parameters, and just use environment variables.